group-coordinator/src/test/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroupTest.java

```diff
@@ -440,12 +442,13 @@ public void testUpdateSubscriptionMetadata() {
         // It should return foo now.
         assertEquals(
             mkMap(
-                mkEntry("foo", new TopicMetadata(fooTopicId, "foo", 1))
+                mkEntry("foo", new TopicMetadata(fooTopicId, "foo", 1, Collections.emptyMap()))
```

Comment:
Should we update `testUpdateSubscriptionMetadata` to include racks for some topics? Otherwise, it seems that we don't really test the newly added logic into `computeSubscriptionMetadata`.